R/coreChecklist.R

```diff
@@ -0,0 +1,18 @@
+# AUTO GENERATED FILE - DO NOT EDIT
+
+coreChecklist <- function(labelClassName=NULL, style=NULL, inputClassName=NULL, inputStyle=NULL, labelStyle=NULL, dashEvents=NULL, options=NULL, className=NULL, values=NULL, fireEvent=NULL, id=NULL, ...) {
+
+    wildcard_names = names(list(...))
+    
+    component <- list(
+        props = list(labelClassName=labelClassName, style=style, inputClassName=inputClassName, inputStyle=inputStyle, labelStyle=labelStyle, dashEvents=dashEvents, options=options, className=className, values=values, fireEvent=fireEvent, id=id, ...),
+        type = 'Checklist',
+        namespace = 'dash_core_components',
+        propNames = c('labelClassName', 'style', 'inputClassName', 'inputStyle', 'labelStyle', 'dashEvents', 'options', 'className', 'values', 'fireEvent', 'id', wildcard_names),
+        package = 'dashCoreComponents'
+        )
+
+    component$props <- filter_null(component$props)
+    
+    structure(component, class = c('dash_component', 'list'))    
+}
```

Comment:
Generated without newline at end of file.

Reply:
The newline isn't required for R scripts.

Reply:
I don't think newline is required for *any* language we use, yet in some cases we actually set our linters to force it. Why?
- It's sometimes useful on the command line (otherwise if you `cat` a file for some reason you'll find your next prompt at the end of the last line of the file, rather than on a new line)
- In some editors, the last line may get intermittently covered up by a horizontal scroll bar
- Github adds 🚫in diffs and nobody wants to annoy Github!

I don't see this as a big deal (unless you have anything to add to this list @T4rk1n), but it's mildly nice and - unless there's a good reason not to - probably easier to just add the newline than to argue about it 😅

Reply:
No, it's not really important, just a pet peeves.

Reply:
Good points, easy to modify the transpiler to add in the newlines next time around!